build.gradle

```diff
@@ -43,7 +31,8 @@ allprojects {
   }
 
   pmd {
-    toolVersion = "6.16.0"
+    toolVersion = "6.25.0"
+    incrementalAnalysis = true
```

Comment:
o didn't know they got this working, nice

Reply:
since gradle 5 point something :upside_down_face: